axelor-sale/src/main/resources/views/AppSale.xml

```diff
@@ -3,10 +3,11 @@
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://axelor.com/xml/ns/object-views http://axelor.com/xml/ns/object-views/object-views_5.0.xsd">
 	
-	<form name="app-sale-config-form" title="App sale" model="com.axelor.apps.base.db.AppSale" canDelete="false" canNew="false" width="large">
+	<form name="app-sale-config-form" title="App sale" model="com.axelor.apps.base.db.AppSale" canDelete="false" canNew="false" width="large" onLoad="action-app-sale-attrs-readonly-manage-sale-order-version-onchange">
```

Comment:
This action-attrs that update a value shouldn't be called on onLoad.
You have to separate into two actions, one for value, another one for readonly (or better, use a javascript expression)